platform/posix/transport/src/plaintext_posix.c

```diff
@@ -126,7 +126,8 @@ int32_t Plaintext_Recv( const NetworkContext_t * pNetworkContext,
     {
         logTransportError( errno );
 
-        /* Check if it was time out. */
+        /* Check if it was time out. Note that for most POSIX implementations,
+         * EAGAIN and EWOULDBLOCK have the same value. */
```

Comment:
Keep in mind that this implies we cannot have full coverage if the POSIX implementation used defines `EAGAIN` and  `EWOULDBLOCK` to have the same value.